src/Symfony/Component/Debug/Tests/Fixtures/ToStringThrower.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Symfony\Component\Debug\Tests\Fixtures;
+
+class ToStringThrower
+{
+    private $exception;
+
+    public function __construct(\Exception $e)
+    {
+        $this->exception = $e;
+    }
+
+    public function __toString()
+    {
+        try {
+            throw $this->exception;
+        } catch (\Exception $e) {
+            return user_error($e, E_USER_ERROR);
```

Comment:
This is why I use `user_error` here: to make sure we do not forget about this alias.